server/build.gradle

```diff
@@ -62,7 +62,7 @@ dependencies {
   api "org.apache.logging.log4j:log4j-core:${versions.log4j}", optional
 
   // repackaged jna with native bits linked against all elastic supported platforms
```

Comment:
Is this comment now redundant?

Reply:
Ah so it is, removed in 5a0d09164e925149ec6e66df62dae0379e263507